src/components/learner-credit-management/data/hooks.js

```diff
@@ -63,18 +63,15 @@ const applySortByToOptions = (sortBy, options) => {
 };
 
 const applyFiltersToOptions = (filters, options) => {
-  const userSearchQuery = filters?.find(filter => filter.id === 'userEmail')?.value;
-  const courseTitleSearchQuery = filters?.find(filter => filter.id === 'courseTitle')?.value;
   const courseProductLineSearchQuery = filters?.find(filter => filter.id === 'courseProductLine')?.value;
```

Comment:
Follow-up from Slack conversation:

> [clarification] in the current state of the (LCM) world, is the courseProductLine filter being used?
> `const courseProductLineSearchQuery = filters?.find(filter => filter.id === 'courseProductLine')?.value;`

[Emily] probably not.

[Adam] even if not used in the current state, since the "Product" column you're keeping in for now isn't exposed in the filters (i.e., disableFilters: true), Markhors next iteration probably will be using it again lolol. So maybe we just leave it in for now with that assumption it will be used again in the near future.